test_app/tests/conftest.py

```diff
@@ -179,6 +179,33 @@ def github_enterprise_team_authenticator(github_enterprise_team_configuration):
     authenticator.delete()
 
 
+@pytest.fixture
+def oidc_configuration():
+    return {
+        "OIDC_ENDPOINT": "https://localhost/api/gateway/callback/github_test/",
+        "OIDC_VERIFY_SSL": True,
+        "KEY": "12345",
+        "SECRET": "abcdefg12345",
+    }
+
+
+@pytest.fixture
+def oidc_authenticator(oidc_configuration):
+    from ansible_base.authentication.models import Authenticator
+
+    authenticator = Authenticator.objects.create(
+        name="Test OIDC Authenticator",
+        enabled=True,
+        create_objects=True,
+        users_unique=False,
+        remove_users=True,
+        type="ansible_base.authentication.authenticator_plugins.oidc",
+        configuration=oidc_configuration,
+    )
+    yield authenticator
+    authenticator.delete()
```

Comment:
We found some issues around the deletion of authentications. If a test associates a user with the authenticator when the test gets back here and tries to delete it you will get an error because there are related users. Most of the authenticator fixtures are incorrectly missing this line right before delete:
```
authenticator.authenticator_user.all().delete()
```

That will remove all associated uses which then allows for the deletion of the authentication. Can you add this line in?